resources/user-agents/browsers/android-browser/android-browser-4-0.json

```diff
@@ -1426,9 +1426,10 @@
           ]
         },
         {
-          "match": "Mozilla/5.0 (#PLATFORM#XOOM 2 Build/*#DEVICE#) applewebkit* (*khtml*like*gecko*) Version/#MAJORVER#.#MINORVER#*Safari*",
+          "match": "Mozilla/5.0 (#PLATFORM#XOOM 2* Build/*#DEVICE#*) applewebkit* (*khtml*like*gecko*) Version/#MAJORVER#.#MINORVER#*Safari*",
```

Comment:
In many cases though, the model number appears in the build string, so I did adjust these in some cases where there was a test with that sort of pattern.